tests/test_check_parallel.py

```diff
@@ -30,6 +31,9 @@
 from pylint.typing import FileItem
 from pylint.utils import LinterStats, ModuleStats
 
+if TYPE_CHECKING:
```

Comment:
Not sure this helps much? If you are running this test then `nodes` will have probably been imported somewhere right?

Reply:
I think we shouldn't reject change because it's only a small improvments. It's better, let's ship it.

Reply:
Fair, I just prefer this conceptually, when it comes in handy in more substantial cases. We can probably trace this to the opposite views we expressed on #8111 :-)